### PR TITLE
build(ingestion-smoke): fix smoke dockerfile

### PR DESCRIPTION
**Pull request by david-leifker**
## Checklist

- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

File: docker/datahub-ingestion-base/smoke.Dockerfile [hunk 1]
```diff
@@ -13,14 +13,16 @@ RUN apt-get update && apt-get install -y \
...
-    sed -i.bak "s/__version__ = \"1\!0.0.0.dev0\"/__version__ = \"$(echo $RELEASE_VERSION|sed s/-/+/)\"/" src/datahub/__init__.py && \
-    cat src/datahub/__init__.py && \
-    cd ../ && \
-    ./gradlew :metadata-ingestion:installAll
+RUN cd /datahub-src && \
+    sed -i.bak "s/__version__ = \"1\!0.0.0.dev0\"/__version__ = \"$(echo $RELEASE_VERSION|sed s/-/+/)\"/" metadata-ingestion/src/datahub/__init__.py && \
+    sed -i.bak "s/__version__ = \"1\!0.0.0.dev0\"/__version__ = \"$(echo $RELEASE_VERSION|sed s/-/+/)\"/" metadata-ingestion-modules/airflow-plugin/src/datahub_airflow_plugin/__init__.py && \
+    cat metadata-ingestion/src/datahub/__init__.py && \
+    ./gradlew :metadata-ingestion:codegen && \
+    pip install file:metadata-ingestion-modules/airflow-plugin#egg=acryl-datahub-airflow-plugin file:metadata-ingestion#egg=acryl-datahub
```
Comment (hsheth2) [inline docker/datahub-ingestion-base/smoke.Dockerfile:27]: I don't think smoke tests need the airflow plugin, but no harm in having it

**Review by hsheth2 — APPROVED** [review]
I don't think oss uses this, so we could also move this to be exclusively in datahub-fork

File: docker/datahub-ingestion-base/smoke.Dockerfile (hunk 1 above)
Comment (david-leifker) [reply docker/datahub-ingestion-base/smoke.Dockerfile:27]: acryl-datahub requires it to install, at least the way it was executing it in installAll

Comment (david-leifker) [reply docker/datahub-ingestion-base/smoke.Dockerfile:27]: there is a github workflow here which references it, should that workflow be removed? @anshbansal ?

Comment (anshbansal) [reply docker/datahub-ingestion-base/smoke.Dockerfile:27]: Without that workflow we won't know whether this dockerfile build breaks or not.

**Pull request merged**